rpcs3/Emu/Cell/Modules/cellVdec.cpp

```diff
@@ -924,7 +924,7 @@ error_code cellVdecClose(ppu_thread& ppu, u32 handle)
 	vdec->abort_decode = true;
 	vdec->in_cmd.push(vdec_cmd(vdec_cmd_type::close, seq_id, cmd_id));
 
-	while (!vdec->ppu_tid)
+	while (!vdec->ppu_tid || vdec->seq_state != sequence_state::closed)
```

Comment:
Not a bug, this code waits to obtain ppu_tid which is zero initialized and set by VDEC thread.

Reply:
How does it relate to what I added?

Reply:
ppu_tid is always written eventually, no need in another condition.

Reply:
ppu_tid is written at thread start.
I want to wait for thread end

Reply:
That's what the syscall is for.

Reply:
I assume the syscall is setting thread abort.
Which is different than abort_decode.

Reply:
No it joins the thread.

Reply:
ah ok. thx

Reply:
but line 940 is redumdant indeed.